src/test/scala/org/broadinstitute/dsde/workbench/sam/api/StatusRouteSpec.scala

```diff
@@ -38,29 +35,24 @@ class StatusRouteSpec extends AnyFlatSpec with Matchers with ScalatestRouteTest
     implicit val patienceConfig = PatienceConfig(timeout = 1 second)
     eventually {
       Get("/status") ~> samRoutes.route ~> check {
-        responseAs[StatusCheckResponse] shouldEqual StatusCheckResponse(true, Map(OpenDJ -> HealthMonitor.OkStatus, Database -> HealthMonitor.OkStatus))
```

Comment:
todo: confirm that it's ok to remove opendj from the status endpoint

Reply:
https://broadinstitute.slack.com/archives/CADM7MZ35/p1663357621764979